db.sh

```diff
@@ -0,0 +1,7 @@
+echo "Checking if all packages are up to date"
```

Comment:
This file should be deleted, I've created the database and such.

Reply:
I think we should keep it, or add it to the wiki. As in copy the contents and add it there under some "Getting Started".